Proposals/XXXX-a-beginners-guide-to-swift-contributions.md

```diff
@@ -0,0 +1,68 @@
+# A Beginner's Guide to Contributing to Swift
+
+* Proposal: [PR#22](https://github.com/SwiftCommunityPodcast/podcast/pull/22)
+
+* Authors: [Andrew Litteken](https://github.com//AndrewLitteken)
+
+* Review Manager: Unassigned
```

Comment:
@BasThomas do you want to be the Review Manager on this one? We haven't quite nailed down what that even means but I think its something like: make sure episodes is source compatible, help proposal author push it forward.